assets/js/components/module/ModuleFooter.js

```diff
@@ -25,17 +25,22 @@ import classNames from 'classnames';
  * Internal dependencies
  */
 import HelpLink from '../HelpLink';
+import { useFeature } from '../../hooks/useFeature';
 
-const ModuleFooter = () => (
-	<div className="mdc-layout-grid__inner">
-		<div className={ classNames( [
-			'mdc-layout-grid__cell',
-			'mdc-layout-grid__cell--span-12',
-			'mdc-layout-grid__cell--align-right',
-		] ) }>
-			<HelpLink />
+const ModuleFooter = () => {
```

Comment:
Just noticed this component is not correctly implemented (not a result of this PR): Can you please reduce it to simply returning `PageFooter`? It should just wrap that component, the same way `DashboardFooter` and `DashboardDetailsFooter` do. The current markup this creates below does not fit correctly with the widget-based module pages.